src/OpenLoco/src/Objects/AirportObject.cpp

```diff
@@ -66,12 +66,12 @@ namespace OpenLoco
             remainingData = remainingData.subspan(ptr - var_1C[i]);
         }
 
-        var_9C = reinterpret_cast<const uint32_t*>(remainingData.data());
+        buildingParts = reinterpret_cast<const AirportBuilding*>(remainingData.data());
         auto* ptr = reinterpret_cast<const uint8_t*>(remainingData.data());
         while (*ptr != 0xFF)
             ptr += 4;
```

Comment:
if you are changing the type to `AirportBuilding` then you'll need to change this one as well.

Reply:
Oh, but this `ptr` is still `const uint8_t*`, right? So there shouldn't be an issue..?